src/TestChild.java

```diff
@@ -5,8 +5,13 @@ public TestChild() {
 		setName("TestChild");
 	}
     
+    // Test additional comment
+    /* Some more comments here*/
     public alotOfArgs(int arg1, int arg2, int arg3, int arg4, int arg5) {
```

Comment:
[Code Smell Detector Automated Comment]
This method has more than the recommended number of arguments.